src/SFA.DAS.AssessorService.Application/Handlers/Certificates/UpdateCertificatesPrintStatusHandler.cs

```diff
@@ -28,14 +28,31 @@ public UpdateCertificatesPrintStatusHandler(ICertificateRepository certificateRe
 
         public async Task<ValidationResponse> Handle(UpdateCertificatesPrintStatusRequest request, CancellationToken cancellationToken)
         {
-            var validationResult = new ValidationResponse();
-
+            var validationResult = new ValidationResponse();            
```

Comment:
You should review your code for 'littering' as well before you commit

Reply:
ok , if you chose hide whitespace changes (just for now) you wont see this.